content/about/notes/1.1/index.md

```diff
@@ -1,32 +1,148 @@
+See [Migrating `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrading-istio#migrating-the-rbacconfig-to-clusterrbacconfig)
+for migration instructions.
+
+- **Identity Provisioning Through SDS**. Provides stronger security with on-node key generation and dynamic certificate rotation without restarting Envoy.
+See [Provisioning Identity through SDS](/docs/tasks/security/auth-sds) for more information.
+
+- **Authorization for TCP Services**. Supports authorization for TCP services in addition to HTTP and gRPC services. 
+See [Authorization for TCP Services](/docs/tasks/security/authz-tcp) for more information.
+
+- **Authorization for End-User Groups**. Allows authorization based on `groups` claim or any list-typed claims in JWT.
+See [Authorization for groups and list claims](/docs/tasks/security/rbac-groups/) for more information.
+
+- **End-User Authentication with Per-Path Requirements**. Allows you to enable or disable JWT authentication based on the request path.
+See [End-user authentication with per-path requirements](/docs/tasks/security/authn-policy/#end-user-authentication-with-per-path-requirements) for 
+more information.
+
+- **External Certificate Management on Ingress Gateway Controller**. Dynamically loads and rotates external certificates.
+
+- **Vault PKI Integration**. Provides stronger security with Vault-protected signing keys and facilitates integration with existing Vault PKIs.
+See [Istio Vault CA Integration](/docs/tasks/security/vault-ca) for more information.
+
+- **Customized (non `cluster.local`) Trust Domains**. Supports organization- or cluster-specific trust domains in the identities.
+
+- TBD: How about adding istio/istio#11667 as well? As this is also a significant feature for security which can enable end user to set
+different CA and Certs for different namespaces.
+
+## Multiclusters
+
+- **Non-Routable L3 Networks**. Enabled using a single Istio control plane in multicluster environments with non-routable
+L3 networks.
+TBD: LINK
+
+- **Multiple Control Planes**. Added support for multiple Istio control planes in support of multicluster environments.
+TBD: LINK
+
+## Policies and telemetry
+
+- **Policy Checks Off By Default**. Changed policy checks to be turned off by default which improves performance for most customer scenarios.
+TBD: LINK
+
+- **Kiali**. Replaced the [Service Graph addon](https://github.com/istio/istio/issues/9066) with [Kiali](https://www.kiali.io) to provide
+a richer visualization experience. See the [Kiali task](/docs/tasks/telemetry/kiali/) for more details.
+
+- **Reduced Overhead**. Added several performance and scale improvements including:
+
+    - Significant reduction in default collection of Envoy-generated statistics (configurable via annotations TBD: LINK)
+
+    - Added load-shedding functionality to Mixer workloads.
+
+    - Improved the protocol between Envoy and Mixer.
+
+- **Control Headers and Routing**. Added the option to create adapters to influence
```

Comment:
@kyessenov did you publish a community-readable design doc (I couldn't find one quickly) ?  Should we have a website task specifically for this item?  It feels like we should.

Reply:
My understanding is that we don't have adapter developer information on the website. We use wiki for that, and there are a couple of guides for this feature. We don't have any check adapters that produce instances yet, so it is not clear how to document that. There is some reference information in the policy APIs but that's about it.

Reply:
Correct, no adapter developer docs on istio.io.

So let's create an issue to track writing user-level docs for this feature so that someone knows what to do once adapters come online with this functionality.

Reply:
https://github.com/istio/istio.io/issues/3673

Reply:
Done.